data/hlint.yaml

```diff
@@ -247,6 +247,8 @@
 
     # TRAVERSABLES
 
+    - warn: {lhs: traverse pure x, rhs: pure x, name: "Traversable law"}
```

Comment:
You should drop the `x` on both sides, so `traverse pure` ==> `pure`.

Reply:
Fixed